site/content/installation/ngf-images/pulling-ngf-image.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Getting the F5 Registry NGINX Plus Gateway Fabric Image
+description: "Learn how to pull an F5 NGINX Plus Gateway Fabric image from the official F5 Docker registry and upload it to your private registry. This guide covers the prerequisites, image tagging, and troubleshooting steps."
+weight: 100
+doctypes: ["install"]
+toc: true
+---
+
+{{<custom-styles>}}
+
+---
+
+## Before you begin
+
+Before you start, you'll need these installed on your machine:
+
+- [Docker v18.09 or higher](https://docs.docker.com/engine/release-notes/18.09/).
+- An NGINX Gateway Fabric subscription. Download both the certificate (*nginx-repo.crt*) and key (*nginx-repo.key*) from the [MyF5 Customer Portal](https://my.f5.com). Keep in mind that an NGINX Plus certificate and key won't work for for the steps in this guide.
+
+---
+
+## Set up Docker for F5 Container Registry
+
+Start by setting up Docker to commungfate with the F5 Container Registry located at `private-registry.nginx.com`. If you're using Linux, follow these steps to create a directory and add your certificate and key:
+
+```shell
+mkdir -p /etc/docker/certs.d/private-registry.nginx.com
+cp <path-to-your-nginx-repo.crt> /etc/docker/certs.d/private-registry.nginx.com/client.cert
+cp <path-to-your-nginx-repo.key> /etc/docker/certs.d/private-registry.nginx.com/client.key
+```
+
+The steps provided are for Linux. For Mac or Windows, consult the [Docker for Mac](https://docs.docker.com/desktop/faqs/general/#how-do-i-add-custom-ca-certificates) or [Docker for Windows](https://docs.docker.com/desktop/faqs/windowsfaqs/#how-do-i-add-custom-ca-certificates) documentation. For more details on Docker Engine security, you can refer to the [Docker Engine Security documentation](https://docs.docker.com/engine/security/).
+
+---
+
+## Pull the image
+
+Next, pull the image you need from `private-registry.nginx.com`. To find the correct image, consult the [Tech Specs guide](https://github.com/nginxinc/nginx-gateway-fabric/blob/main/CHANGELOG.md).
```

Comment:
```suggestion
Next, pull the image you need from `private-registry.nginx.com`. To find the correct image, consult the [Technical Specifications][(https://github.com/nginxinc/nginx-gateway-fabric/blob/main/CHANGELOG.md](https://github.com/nginxinc/nginx-gateway-fabric#technical-specifications.
```
This edit suggestion follows the conversation Saylor started in another thread.

Reply:
updating